opdracht-1.js

```diff
@@ -0,0 +1,80 @@
+// Opdracht 1 - Array Methoden
+
+
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+// via de map methode een nieuwe array gemaakt met alle tv-types
+const tvTypes = inventory.map((inventory) => {return inventory.type});
+console.log(tvTypes);
+
+//     Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+// door de filter methode te gebruiken - en als de originalStock - sold === 0 --> dan is het uitverkocht
+
+const soldOutTvs = inventory.filter((inventory) => {return (inventory.originalStock - inventory.sold )=== 0});
```

Comment:
Werkt prima zo, maar het kan nog iets simpeler 😉 Kijk ook uit dat je niet conflicteren variabele namen kiest
```suggestion
const soldOutTvs = inventory.filter(tv => tv.originalStock === tv.sold);
```